fpga/Makefile

```diff
@@ -120,9 +135,19 @@ debug-bitstream: $(build_dir)/obj/post_synth.dcp
 			$(build_dir)/debug_obj \
 			$(fpga_common_script_dir)
 
+MCS_FILE := $(build_dir)/obj/$(MODEL).mcs
+$(MCS_FILE): $(BIT_FILE)
+	cd $(build_dir); vivado \
+		-nojournal -mode batch \
+		-source $(fpga_common_script_dir)/write_cfgmem.tcl \
+		-tclargs $(BOARD) $@ $^ \
+
+.PHONY: mcs
+mcs: $(MCS_FILE)
+
 #########################################################################################
 # general cleanup rules
 #########################################################################################
 .PHONY: clean
 clean:
-	rm -rf $(gen_dir)
```

Comment:
In Chipyard we have this `clean` target "clean" everything (hence the `gen_dir` being deleted). I wouldn't be opposed to adding a new target like `clean-specific` (or a better name) that just deletes the sources related to this config.

Reply:
I agree with your suggestion that It would be better to have a new target like `clean-specific` for a certain config!